docs/source/noise/column_noise.rst

```diff
@@ -248,6 +248,32 @@ It takes two parameters:
       The second value in the list is the corresponding probability for the second digit, and so on.
     - [0.04, 0.04, 0.20, 0.36, 0.36]
 
+Swap month and day
+------------------
+
+Swap month and day is a noise type that only applies to dates. It occurs when someone swaps the month and day to be in the incorrect position (e.g., December 8, 2022 would be listed in MM/DD/YYYY format as 08/12/2022).
+
+This noise type is called :code:`swap_month_and_day` in the configuration. It takes two parameters:
+
+.. list-table:: Parameters to the swap_month_and_day noise type
```

Comment:
This noise function does not have a `token_probability` -- it has a `cell_probability` only, which is directly the probability that the month and day get swapped. This is simpler than some of the other noise types, because there is only one "place" in a date where the month and day can be swapped (contrast with e.g. typos which can happen multiple times within a single string).

Reply:
Oops yes I was confused with this section - will fix this, thanks!